### DISCUSSION TITLE
Disable logging on gateway.

### DISCUSSION THREAD

**Discussion by yetanothern** [apiman/apiman#2308] [Q&A] [answered]
Is it possible to fully disable logging on gateways?
In `apiman.properties` I have `apiman-gateway.logger-factory=io.apiman.common.logging.impl.NoOpLoggerFactory`.
According to https://www.apiman.io/apiman-docs/core/2.2.3.Final/development/Guide.html#_logging it's already null, but I have data in gw_requests table in gateway database.

**Comment by msavy** [comment, marked as answer]
This will be easier in Apiman 3. There might be some components logging outside of the logging framework(s).
